packages/utils/types/Config.ts

```diff
@@ -0,0 +1,15 @@
+export interface Config extends Object {
+	item?: {
+		name: string;
+	};
+	topScope?: string[];
+	configName?: string;
+	merge: object;
```

Comment:
```suggestion
	merge: string|string[];
```
We have now changed the `merge` functionality at #797, and this is the type we need now.